tests/Feature/Commands/Application/DataTransferObjectMakeCommandTest.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+namespace Tests\Feature\Commands;
+
+use Illuminate\Support\Facades\Artisan;
+use Illuminate\Support\Facades\File;
+use Illuminate\Support\Str;
+use KoalaFacade\DiamondConsole\Exceptions\FileAlreadyExistException;
+
+it(description: 'can generate new DTO')
+    ->tap(function () {
+        $filePath = app_path(path: 'DataTransferObjects/Post/PostData.php');
```

Comment:
Can you change the `app_path()` with `applicationPath()` based on [Pest.php](https://github.com/KoalaFacade/diamond-console/blob/481fbe5c11949a409bf44852aa61fbf26b3f2b78/tests/Pest.php#L85)

Reply:
ight